habitat-lab/habitat/gui/image_framebuffer_drawer.py

```diff
@@ -21,29 +21,79 @@ def __init__(self, max_width=1440, max_height=1440):
             mn.gl.Framebuffer.ColorAttachment(0), self.texture, 0
         )
 
-    def draw(self, pixel_data, dest_x, dest_y):
+    # pad images to avoid error in Magnum ImageView code.
+    def _pad_image_tensor_for_draw_requirements(self, tensor):
+        # todo: investigate Magnum ImageView2D to understand actual requirements here.
+        required_width_multiple = 4
```

Comment:
This "4" guess is a hack. I tested locally and saw a requirement for 4-pixel alignment. But this probably varies based on a lot of factors, e.g. pixel format or specific GPU. This is fine for now and we can always increase to 8 or 16 if other folks hit errors when using debug images and this code.